Android/app/src/main/res/layout/statistics_item.xml

```diff
@@ -0,0 +1,32 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/cardViewParent"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.cardview.widget.CardView
```

Comment:
Also a nit and it totally works as is but feels like we aren't using any of the benefits of a cardview and this could probably just be a textview without the parent CardView layout

Reply:
Another good catch! I just removed the elevation last night to better match figma.